### PR TITLE
change dates and add Bosworth, Rene, Will

### PR DESCRIPTION
**Pull request by adamjonas**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -245,6 +245,28 @@ <h4>Andrew Poelstra</h4>
...
+                  </a>
+                </li>
+                <li class="list-inline-item">
+                  <a target="_blank" href="https://twitter.com/alexbosworth">
+                    <i class="fa fa-twitter"></i>
+                  </a>
+                </li>
+              </ul>
+              <div class="bio">
+                Alex is a lightning infrastructure lead at Lightning Labs. Previously, he created yalls.org, submarineswaps.org, and helped develop a blockchain project used to provide transparency for the Royal Mint of the U.K.
```
Comment (jnewbery) [inline index.html]: s/a lightning infrastructure lead/the lightning infrastructure lead/

File: index.html [hunk 2]
```diff
@@ -543,6 +543,72 @@ <h4>Johan Torås Halseth</h4>
...
+                  </a>
+                </li>
+                <li class="list-inline-item">
+                  <a target="_blank" href="https://twitter.com/carl_dong">
+                    <i class="fa fa-twitter"></i>
+                  </a>
+                </li>
+              </ul>
+              <div class="bio">
+                Carl is a developer at Chaincode Labs where he contributes to rust-bitcoin and Bitcoin. He is excited about systems engineering and incentive-compatible uses of permissionless money.
```
Comment (jnewbery) [inline index.html]: a/rust-bitcoin and Bitcoin./rust-bitcoin and Bitcoin Core./

File: index.html (hunk 1 above)
Comment (jnewbery) [reply index.html]: Also, I'm not sure if working on the Royal Mint project is what he'd choose to put in his bio. I think I'd just leave it at "Previously, he created yalls.org, htlc.me and submarineswaps.org"

Comment (adamjonas) [reply index.html]: https://lightning.engineering/team.html

**Pull request merged**